azure-pipelines-templates/download-install-cmake.yml

```
@@ -0,0 +1,20 @@
+# Copyright (c) .NET Foundation and Contributors
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)